src/Javis.jl

```diff
@@ -233,6 +233,12 @@ Streaming to Twitch or other platforms are not yet supported.
     - Can be used if there are errors with ffmpeg. Defaults to panic:
     All other options are described here: https://ffmpeg.org/ffmpeg.html
 - `rescale_factor::Float64` factor to which the frames should be rescaled for faster rendering
+- `postprocess_frame::Function` function that is applied to the imagematrix of each frame after they have been computed 
+takes as argument postprocess_frame(frame_image, frame, frames), useful to apply a postprocessing e.g. blur to all or some 
```

Comment:
I would write `frame_image, frame, frames` so without the function name and in the code ticks